plugins/logo/README.md

```diff
@@ -67,21 +89,24 @@ Input, Output, Mark to write to Siemens Logo
 [myroom]
     [[status_I1]]
         typ = bool
-        logo_read = I1
+        logo_read@logo1 = I1    # read the Input I1 from Logo-Instance 'logo1'
```

Comment:
Do we want to explain the multi-instance configuration style in all plugins supporting it? This should be in the common plugin (configuration) documentation in the wiki, I think.